src/Microsoft.ServiceFabric.Actors/Runtime/ActorGarbageCollectionSettings.cs

```diff
@@ -22,9 +22,9 @@ public ActorGarbageCollectionSettings()
         }
 
         /// <summary>
-        /// Copy constructor.
+        /// Initializes a new instance of the ActorGarbageCollectionSettings class.
```

Comment:
Should be something like "Initializes a new instance of the ActorGarbageCollectionSettings class with the values of the input argument."